src/redux/actionTypes/index.js

```diff
@@ -17,4 +17,20 @@ export default _.mapKeys([
   'MESSAGES_FEEDBACK_RECEIVED',
   'MESSAGES_INIT_TTS_VOICES',
   'MESSAGES_GET_HISTORY_FROM_SERVER',
+  'SETTINGS_SET_USER_SETTINGS',
```

Comment:
I think renaming SETTINGS_SET_USER_SETTINGS to SETTINGS_SAVE_USER_SETTINGS would make more sense?

Reply:
I think Set(Get is being used) is also fine, isn't it? For 'GET' specific settings, we are using one API for fetching and updating. Creating separate action creators, reducers for it, won't matter much

Reply:
One would be to get the settings from server, which one would be used to send them to server?

Reply:
'SETTINGS_SET_USER_SETTINGS'

Reply:
I might sound stupid right now, SETTINGS_SET_USER_SETTINGS is to send the settings to the server, what would you do to initialize the store. Coz I see not api that uses this url ```/aaa/listUserSettings.json?``` which is used to fetch user settings from server.

Reply:
API.actions.js Line number 424

Reply:
`chat.susi.ai/src/apis/index.js ` line `39` function does that. getUserSettings is used to fetch the user settings.

Reply:
Wait, I can't see it in your PR.. oh.. it was probably implemented by Akshat sir. So do we have an action to call that api or do you plan to call it from the component directly? @fragm3

Reply:
Action